browser/src/view/components/Navbar/NavigationMenu.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FiHome, FiLayers, FiSettings, FiLogOut, FiCircle } from 'react-icons/fi';
+import './NavigationMenu.css';
+
+export const NavigationMenu = () => {
+  return (
+    <>
+      <div className='navbar'>
```

Comment:
You should use semantic HTML here.  There's a `nav` tag that better fits navigation components.  This will help tremendously when it comes to making our application accessible.

Also, we'll probably want to design an interface in this component that allows a consumer to provide the nav items as props.  I'd encourage you to conceptualize a way for that to happen.  Remember that the navigation component doesn't need to know anything about our application (besides style and such) -- it can be handled as if it were a common component.

Reply:
`prop-types` can help you define the interface a consumer can use.  For example:

```js
NavigationMenu.propTypes = {
  links: PropTypes.arrayOf(PropTypes.shape({
    href: PropTypes.string.isRequired,
    isExternal: PropTypes.bool.isRequired,
  }).isRequired).isRequired
};
```

You could define it on the backend.  I think we can do it on the frontend for now in the component that's rendering the nav.  We can figure out better strategies later.

Reply:
Yes it would.